voxblox/src/integrator/tsdf_integrator.cc

```diff
@@ -229,11 +230,24 @@ float TsdfIntegratorBase::computeDistance(const Point& origin,
 // Thread safe.
 float TsdfIntegratorBase::getVoxelWeight(const Point& point_C) const {
   if (config_.use_const_weight) {
-    return 1.0f;
+    return config_.voxel_weight_factor;
```

Comment:
What is the `_factor` part?

Reply:
Got the info from below although can we assume that the const weight is the same as the factor?

Reply:
true, so to be correct, `factor` only applies to the lower part, where it is a factor to the inverse squared depth. but as we have `use_constant_weight` disabled by default, I thought that it's probably better to call it factor.